FateGrandAutomata/AndroidImpl.cs

```diff
@@ -55,6 +62,14 @@ public void MessageBox(string Title, string Message)
             });
         }
 
+        public void Highlight(Region Region, double Timeout)
```

Comment:
How about replacing double with TimeUnit everywhere? That way you won't make the mistake again of not knowing what time unit it is.